velero-csi-plugin/csi_restorer.go

```diff
@@ -63,16 +66,35 @@ func (p *CSIRestorer) Execute(input *velero.RestoreItemActionExecuteInput) (*vel
 		return nil, errors.Errorf("Could not find volume snapshot name on PVC")
```

Comment:
I'm guessing we don't want to return an error here, since not all PVCs are expected to have a CSI snapshot. Probably just debug/info log and return early.